web-ui/src/main/resources/catalog/components/viewer/gfi/FeaturesLoader.js

```diff
@@ -252,7 +252,7 @@
 
       var columns = Object.keys(features[0].getProperties()).map(function (x) {
         return {
-          field: x,
+          field: x.toLowerCase(),
```

Comment:
To be done for embedded feature catalogue too. See https://github.com/GeoCat/core-geonetwork/pull/69